nuxeo-features/nuxeo-thumbnail/src/main/java/org/nuxeo/ecm/platform/thumbnail/io/ThumbnailJsonEnricher.java

```diff
@@ -80,7 +81,7 @@ public void write(JsonGenerator jg, DocumentModel document) throws IOException {
         jg.writeStringField(THUMBNAIL_URL_LABEL,
                 String.format(THUMBNAIL_URL_PATTERN, ctx.getBaseUrl().replaceAll("/$", ""),
                         document.getRepositoryName(), document.getId(),
-                        URLEncoder.encode(document.getChangeToken(), "UTF-8")));
+                        URLEncoder.encode(defaultString(document.getChangeToken()), "UTF-8")));
```

Comment:
commons-lang3 dependency must be added to the pom.xml

Reply:
No need, it's an indirect dependency already